cmd/event-remove.go

```diff
@@ -32,6 +32,18 @@ var (
 			Name:  "force",
 			Usage: "force removing all bucket notifications",
 		},
+		cli.StringFlag{
```

Comment:
Need to update the license header to include 2020

Reply:
done